hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/FSHLog.java

```diff
@@ -1317,8 +1317,12 @@ public void run() {
           Throwable lastException = null;
           try {
             Trace.addTimelineAnnotation("syncing writer");
+            long unSyncedFlushSeq = highestUnsyncedSequence;
```

Comment:
Can we avoid introducing unSyncedFlushSeq (use highestUnsyncedSequence directly) ?

Reply:
Could change during the sync.

Reply:
yeah ， may change during writer.sync